.github/workflows/blog-links.yaml

```diff
@@ -0,0 +1,35 @@
+name: Blog Links
+
+on:
+  schedule:
+    - cron: "20 7 * * *"
+
+jobs:
+  check-links:
+    if: github.repository == 'canonical-web-and-design/ubuntu.com'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout master
+        uses: actions/checkout@v2
+
+      - name: Install linkchecker
+        run: sudo pip install LinkChecker
+
+      - name: Run linkchecker
+        run: |
+          linkchecker --recursion-level=2 \
+          	--check-extern --no-warnings https://ubuntu.com/blog \
+          	--no-follow-url '!https:\/\/ubuntu.com\/blog' \
+          	--ignore-url https://res.cloudinary.com \
+          	--ignore-url q_auto \
+          	--ignore-url fl_sanitize \
+          	--ignore-url c_fill \
+          	--ignore-url e_sharpen \
+          	--ignore-url w_[0-9]* \
+          	--ignore-url h_[0-9]* \
+          	--ignore-url https://ubuntu.com/security
+
+      - name: Send message on failure
+        if: failure()
+        run: curl -X POST -F "workflow=${GITHUB_WORKFLOW}" -F "repo_name=${GITHUB_REPOSITORY}" -F "action_id=${GITHUB_RUN_ID}" ${{ secrets.BOT_URL }}?room=test-bot-channel
```

Comment:
This channel should be updated before merging

Reply:
@anthonydillon which channel should blog links errors be sent to?

Reply:
Marketing, probably as it's mostly there content

Reply:
Blog and live-links actions are going to Marketing now. 
Security goes to the CVE & USN channel.